_std/defines/trader.dm

```diff
@@ -0,0 +1,4 @@
+// Rarities to determine item amounts in traders
+#define ITEM_COMMON 1
+#define ITEM_UNCOMMON 2
+#define ITEM_RARE 3
```

Comment:
Proobably best to rename these to `TRADER_RARITY_COMMON` or similar to distinguish them from the `ITEM_RARITY_COMMON` defines which already exist.